client/src/containers/AuthContainer.tsx

```diff
@@ -29,72 +41,82 @@ function TabPanel(props: TabPanelProps) {
       aria-labelledby={`full-width-tab with index: ${index}`}
       {...rest}
     >
-      {value === index && (
-        <Box>
-          <div>{children}</div>
-        </Box>
-      )}
+      {value === index && <Box sx={{ p: 3 }}>{children}</Box>}
     </div>
   );
 }
 
 function AuthContainer() {
-  const paperStyle = {
-    height: "100%",
-    borderRadius: "20px",
-  };
-
   const [tabIndex, setTabIndex] = useState(0);
 
-  const handleChange = (event: React.ChangeEvent<{}>, newValue: number) => {
+  const handleChange = (event: React.SyntheticEvent, newValue: number) => {
     setTabIndex(newValue);
   };
 
-  const matches = useMediaQuery("(min-width: 1080px)");
+  const matches = useMediaQuery(theme.breakpoints.down("md"));
 
   return (
     <div className="AuthContainer">
-      <Grid
-        container
-        justifyContent={matches ? "flex-end" : "center"}
-        style={{ padding: matches ? "20px" : 0 }}
-      >
-        <Grid item md={7} lg={8} xl={9}>
-          <Box
-            style={{
-              textAlign: "center",
-              paddingTop: matches ? "100px" : "0px",
-            }}
-            fontFamily="Verdana"
-            color="white"
-          >
-            <h1 style={{ fontSize: matches ? "500%" : "200%" }}>UOFTBOARD</h1>
-            <h1>A place to connect</h1>
-          </Box>
-        </Grid>
-
-        <Grid item md={5} lg={4} xl={3}>
-          <Paper elevation={5} style={paperStyle}>
-            <Tabs
-              value={tabIndex}
-              onChange={handleChange}
-              aria-label="Login and Signup tabs"
-              variant="fullWidth"
-              textColor="primary"
+      <ThemeProvider theme={theme}>
+        <CssBaseline />
+        <Grid
+          container
+          component="main"
+          justifyContent="flex-end"
+          alignItems="center"
+          sx={{ height: "100vh", px: "40px" }}
+        >
+          <Grid item hidden={matches} xs={false} md={8}>
+            <Box
+              sx={{
+                display: "flex",
+                flexDirection: "column",
+                textAlign: "center",
+                alignItems: "center",
+              }}
             >
-              <Tab label="Login" data-testid="LogInTabButton" />
-              <Tab label="Sign Up" data-testid="SignUpTabButton" />
-            </Tabs>
-
-            <TabPanel value={tabIndex} index={0}>
-              <Login handleChange={handleChange} />
-            </TabPanel>
-            <TabPanel value={tabIndex} index={1}>
-              <SignUp handleChange={handleChange} />
-            </TabPanel>
-          </Paper>
+              <Box bgcolor="rgba(83, 102, 133, 0.6)" p="40px" px="80px">
+                <Typography color="white" fontFamily="Verdana" variant="h2">
```

Comment:
typography would be a good theme variable.

Reply:
Decided not to use Verdana, so theme for Typography is currently undecided.

Reply:
I think we can stick with the theme default, we don't need to declare fontFamily in any of our typography calls.